.github/workflows/build-python-multi.yaml

```diff
@@ -27,9 +27,11 @@ on:
       ecr-repository-code:
         type: string
         required: true
+        default: 860641649575
```

Comment:
not sure that the default is ever used when required is set to true 
**https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault **

Reply:
fixed